dll/win32/shlwapi/propbag.cpp

```diff
@@ -0,0 +1,336 @@
+/*
+ * PROJECT:     ReactOS Shell
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Implement shell property bags
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include "precomp.h"
+#include <shlwapi.h>
+#include <shlwapi_undoc.h>
+
+WINE_DEFAULT_DEBUG_CHANNEL(shell);
+
+// Dynamic structure array (DSA); See COMCTL32
+struct _DSA
```

Comment:
This struct does not seem to be used here?

Reply:
`HDSA` is a pointer to `struct _DSA`. This is the definition.

Reply:
Deleted.